scrapers.js

```diff
@@ -61,7 +63,8 @@ let scrapers = [
     }
   },
   {
-    url: 'https://opendata.arcgis.com/datasets/8840fd8ac1314f5188e6cf98b525321c_0.csv',
+    url:
+      'https://opendata.arcgis.com/datasets/8840fd8ac1314f5188e6cf98b525321c_0.csv',
```

Comment:
Still dropping these guys down to the next line, can we prevent that?

Reply:
oh, sorry didn't notice that

Yeah, give me one minute.